### PR TITLE
feat(networks,governance): add Linea to cross chain governance

### PR DESCRIPTION
**Pull request by clemsos**
# Description

This adds support for Linea to the cross-chain governance process.

TODO:

- [x] add `governanceBridge` section to the network package
- [x] ~~deploy safe delay module master copies~~ support linea in Gnosis Guild core lib  https://github.com/gnosisguild/zodiac/pull/163
- [x] deploy delayMod instance 
- [x] verify the delayMod
- [x] deploy the connextMod for the multisig
- [x] verify the connextMod
- [x]  add the modules addresses to the `governanceBridge` section with the deployed modules address
- [x] enable modules correctly in SAFE 

# Issues

Fixes #
Refs #13463

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

**Comment by unlock-deployer** [comment]
### packages/networks/src/networks/linea.ts

The setup is successful :

- ✅ all mandatory keys are present
- ✅ Contract PublicLockLatest at 0x70cBE5F72dD85aA634d07d2227a421144Af734b3 is verified
- ✅ Contract Unlock at 0x70B3c9Dd9788570FAAb24B92c3a57d99f8186Cc7 is verified
- ✅ Contract KeyManager at 0x338b1f296217485bf4df6CE9f93ab4C73F72b57D is verified
- ✅ Contract onKeyPurchaseHook_PASSWORD at 0x6878Ae3c863f6Ebd27B47C02F6B32aAC8B0BA07E is verified
- ✅ Contract onKeyPurchaseHook_CAPTCHA at 0x8c5D54B2CAA4C2D08B0DDF82a1e6D2641779B8EC is verified
- ✅ Contract onKeyPurchaseHook_GUILD at 0xaE8F3F0826A39122401ED634f0a5C19549331432 is verified
- ✅ Contract onKeyPurchaseHook_PROMOCODE at 0xCD9C9b40D757b56359e19563203D3bc64089638d is verified
- ✅ Contract onKeyPurchaseHook_PASSWORD_CAPPED at 0xD925Ac2887Ba4372849F0fd64217A6749552bb21 is verified
- ✅ Subgraph up and running
- ✅ Unlock ownership correctly set to the multisig

We have found the following errors :

- ❌ Contract onKeyPurchaseHook_PROMO_CODE_CAPPED at 0x4Bf912519549DF750002814a2DcE7184b3971F06 is not verified
- ❌ Missing signers: 0x2785f2a3DDaCfDE5947F1A9D6c878CCD7F885400,0x7A23608a8eBe71868013BDA0d900351A83bb4Dc2,0x8de33D8204929ceb2F7AA6299d0643a7f6664c9b

Some verification calls have failed  :

- ❌ Could not fetch Unlock source code from block explorer, undefined
- ❌ Could not fetch PublicLock source code from block explorer, TypeError: Cannot read properties of undefined (reading 'substring')

Some additonal warnings  :

- ⚠️ Contract PublicLockPrevious is missing

**Comment by unlock-deployer** [comment]
### packages/networks/src/networks/linea.ts

The setup is successful :

- ✅ all mandatory keys are present
- ✅ Contract PublicLockLatest at 0x70cBE5F72dD85aA634d07d2227a421144Af734b3 is verified
- ✅ Contract Unlock at 0x70B3c9Dd9788570FAAb24B92c3a57d99f8186Cc7 is verified
- ✅ Contract KeyManager at 0x338b1f296217485bf4df6CE9f93ab4C73F72b57D is verified
- ✅ Contract onKeyPurchaseHook_PASSWORD at 0x6878Ae3c863f6Ebd27B47C02F6B32aAC8B0BA07E is verified
- ✅ Contract onKeyPurchaseHook_PROMOCODE at 0xCD9C9b40D757b56359e19563203D3bc64089638d is verified
- ✅ Contract onKeyPurchaseHook_PASSWORD_CAPPED at 0xD925Ac2887Ba4372849F0fd64217A6749552bb21 is verified
- ✅ Contract onKeyPurchaseHook_CAPTCHA at 0x8c5D54B2CAA4C2D08B0DDF82a1e6D2641779B8EC is verified
- ✅ Contract onKeyPurchaseHook_GUILD at 0xaE8F3F0826A39122401ED634f0a5C19549331432 is verified
- ✅ Subgraph up and running
- ✅ Unlock ownership correctly set to the multisig

We have found the following errors :

- ❌ Verification failed for onKeyPurchaseHook_PROMO_CODE_CAPPED at 0x4Bf912519549DF750002814a2DcE7184b3971F06: Contract source code not verified
- ❌ Unlock source code at 0x70B3c9Dd9788570FAAb24B92c3a57d99f8186Cc7 does not match packaged UnlockV13
- ❌ PublicLock source code at 0x70cBE5F72dD85aA634d07d2227a421144Af734b3 does not match packaged PublicLockV14
- ❌ Missing signers: 0x2785f2a3DDaCfDE5947F1A9D6c878CCD7F885400,0x7A23608a8eBe71868013BDA0d900351A83bb4Dc2,0x8de33D8204929ceb2F7AA6299d0643a7f6664c9b




Some additonal warnings  :

- ⚠️ Contract PublicLockPrevious is missing
- ⚠️ Contract onKeyPurchaseHook_CUSTOM_CONTRACT is missing
- ⚠️ Contract onKeyPurchaseHook_GITCOIN is missing
- ⚠️ Contract onKeyPurchaseHook_ADVANCED_TOKEN_URI is missing

**Comment by clemsos** [comment]
> Some verification calls have failed :
> 
> * ❌ Could not fetch Unlock source code from block explorer, undefined
> * ❌ Could not fetch PublicLock source code from block explorer, TypeError: Cannot read properties of undefined (reading 'substring')

It happens that Linea was still on v13...

**Comment by unlock-deployer** [comment]
### packages/networks/src/networks/linea.ts

The setup is successful :

- ✅ all mandatory keys are present
- ✅ Contract PublicLockLatest at 0x70cBE5F72dD85aA634d07d2227a421144Af734b3 is verified
- ✅ Contract Unlock at 0x70B3c9Dd9788570FAAb24B92c3a57d99f8186Cc7 is verified
- ✅ Contract KeyManager at 0x338b1f296217485bf4df6CE9f93ab4C73F72b57D is verified
- ✅ Contract onKeyPurchaseHook_PASSWORD at 0x6878Ae3c863f6Ebd27B47C02F6B32aAC8B0BA07E is verified
- ✅ Contract onKeyPurchaseHook_PROMOCODE at 0xCD9C9b40D757b56359e19563203D3bc64089638d is verified
- ✅ Contract onKeyPurchaseHook_PASSWORD_CAPPED at 0xD925Ac2887Ba4372849F0fd64217A6749552bb21 is verified
- ✅ Contract onKeyPurchaseHook_CAPTCHA at 0x8c5D54B2CAA4C2D08B0DDF82a1e6D2641779B8EC is verified
- ✅ Contract onKeyPurchaseHook_GUILD at 0xaE8F3F0826A39122401ED634f0a5C19549331432 is verified
- ✅ Subgraph up and running
- ✅ Unlock ownership correctly set to the multisig

We have found the following errors :

- ❌ Verification failed for onKeyPurchaseHook_PROMO_CODE_CAPPED at 0x4Bf912519549DF750002814a2DcE7184b3971F06: Contract source code not verified
- ❌ Unlock source code at 0x70B3c9Dd9788570FAAb24B92c3a57d99f8186Cc7 does not match packaged UnlockV13
- ❌ PublicLock source code at 0x70cBE5F72dD85aA634d07d2227a421144Af734b3 does not match packaged PublicLockV14
- ❌ Missing signers: 0x2785f2a3DDaCfDE5947F1A9D6c878CCD7F885400,0x7A23608a8eBe71868013BDA0d900351A83bb4Dc2,0x8de33D8204929ceb2F7AA6299d0643a7f6664c9b




Some additonal warnings  :

- ⚠️ Contract PublicLockPrevious is missing
- ⚠️ Contract onKeyPurchaseHook_CUSTOM_CONTRACT is missing
- ⚠️ Contract onKeyPurchaseHook_GITCOIN is missing
- ⚠️ Contract onKeyPurchaseHook_ADVANCED_TOKEN_URI is missing

**Comment by unlock-deployer** [comment]
### packages/networks/src/networks/linea.ts

The setup is successful :

- ✅ all mandatory keys are present
- ✅ Contract PublicLockLatest at 0x70cBE5F72dD85aA634d07d2227a421144Af734b3 is verified
- ✅ Contract Unlock at 0x70B3c9Dd9788570FAAb24B92c3a57d99f8186Cc7 is verified
- ✅ Contract KeyManager at 0x338b1f296217485bf4df6CE9f93ab4C73F72b57D is verified
- ✅ Contract onKeyPurchaseHook_PASSWORD at 0x6878Ae3c863f6Ebd27B47C02F6B32aAC8B0BA07E is verified
- ✅ Contract onKeyPurchaseHook_PROMOCODE at 0xCD9C9b40D757b56359e19563203D3bc64089638d is verified
- ✅ Contract onKeyPurchaseHook_PASSWORD_CAPPED at 0xD925Ac2887Ba4372849F0fd64217A6749552bb21 is verified
- ✅ Contract onKeyPurchaseHook_CAPTCHA at 0x8c5D54B2CAA4C2D08B0DDF82a1e6D2641779B8EC is verified
- ✅ Contract onKeyPurchaseHook_GUILD at 0xaE8F3F0826A39122401ED634f0a5C19549331432 is verified
- ✅ Subgraph up and running
- ✅ Unlock ownership correctly set to the multisig

We have found the following errors :

- ❌ Verification failed for onKeyPurchaseHook_PROMO_CODE_CAPPED at 0x4Bf912519549DF750002814a2DcE7184b3971F06: Contract source code not verified
- ❌ Unlock source code at 0x70B3c9Dd9788570FAAb24B92c3a57d99f8186Cc7 does not match packaged UnlockV13
- ❌ PublicLock source code at 0x70cBE5F72dD85aA634d07d2227a421144Af734b3 does not match packaged PublicLockV14
- ❌ Missing signers: 0x2785f2a3DDaCfDE5947F1A9D6c878CCD7F885400,0x7A23608a8eBe71868013BDA0d900351A83bb4Dc2,0x8de33D8204929ceb2F7AA6299d0643a7f6664c9b




Some additonal warnings  :

- ⚠️ Contract PublicLockPrevious is missing
- ⚠️ Contract onKeyPurchaseHook_CUSTOM_CONTRACT is missing
- ⚠️ Contract onKeyPurchaseHook_GITCOIN is missing
- ⚠️ Contract onKeyPurchaseHook_ADVANCED_TOKEN_URI is missing

**Comment by unlock-deployer** [comment]
### packages/networks/src/networks/linea.ts

The setup is successful :

- ✅ all mandatory keys are present
- ✅ Contract PublicLockLatest at 0x70cBE5F72dD85aA634d07d2227a421144Af734b3 is verified
- ✅ Contract Unlock at 0x70B3c9Dd9788570FAAb24B92c3a57d99f8186Cc7 is verified
- ✅ Contract KeyManager at 0x338b1f296217485bf4df6CE9f93ab4C73F72b57D is verified
- ✅ Contract onKeyPurchaseHook_PASSWORD at 0x6878Ae3c863f6Ebd27B47C02F6B32aAC8B0BA07E is verified
- ✅ Contract onKeyPurchaseHook_PROMOCODE at 0xCD9C9b40D757b56359e19563203D3bc64089638d is verified
- ✅ Contract onKeyPurchaseHook_PASSWORD_CAPPED at 0xD925Ac2887Ba4372849F0fd64217A6749552bb21 is verified
- ✅ Contract onKeyPurchaseHook_CAPTCHA at 0x8c5D54B2CAA4C2D08B0DDF82a1e6D2641779B8EC is verified
- ✅ Contract onKeyPurchaseHook_GUILD at 0xaE8F3F0826A39122401ED634f0a5C19549331432 is verified
- ✅ Subgraph up and running
- ✅ Unlock ownership correctly set to the multisig

We have found the following errors :

- ❌ Verification failed for onKeyPurchaseHook_PROMO_CODE_CAPPED at 0x4Bf912519549DF750002814a2DcE7184b3971F06: Contract source code not verified
- ❌ Unlock source code at 0x70B3c9Dd9788570FAAb24B92c3a57d99f8186Cc7 does not match packaged UnlockV13
- ❌ PublicLock source code at 0x70cBE5F72dD85aA634d07d2227a421144Af734b3 does not match packaged PublicLockV14
- ❌ Missing signers: 0x2785f2a3DDaCfDE5947F1A9D6c878CCD7F885400,0x7A23608a8eBe71868013BDA0d900351A83bb4Dc2,0x8de33D8204929ceb2F7AA6299d0643a7f6664c9b




Some additonal warnings  :

- ⚠️ Contract PublicLockPrevious is missing
- ⚠️ Contract onKeyPurchaseHook_CUSTOM_CONTRACT is missing
- ⚠️ Contract onKeyPurchaseHook_GITCOIN is missing
- ⚠️ Contract onKeyPurchaseHook_ADVANCED_TOKEN_URI is missing

**Comment by clemsos** [comment]
need to be rebased once https://github.com/unlock-protocol/unlock/pull/14636 is merged

**Comment by unlock-deployer** [comment]
### packages/networks/src/networks/linea.ts

The setup is successful :

- ✅ all mandatory keys are present
- ✅ Contract PublicLockLatest at 0x70cBE5F72dD85aA634d07d2227a421144Af734b3 is verified
- ✅ Contract Unlock at 0x70B3c9Dd9788570FAAb24B92c3a57d99f8186Cc7 is verified
- ✅ Contract KeyManager at 0x338b1f296217485bf4df6CE9f93ab4C73F72b57D is verified
- ✅ Contract onKeyPurchaseHook_PASSWORD at 0x6878Ae3c863f6Ebd27B47C02F6B32aAC8B0BA07E is verified
- ✅ Contract onKeyPurchaseHook_PROMOCODE at 0xCD9C9b40D757b56359e19563203D3bc64089638d is verified
- ✅ Contract onKeyPurchaseHook_PASSWORD_CAPPED at 0xD925Ac2887Ba4372849F0fd64217A6749552bb21 is verified
- ✅ Contract onKeyPurchaseHook_CAPTCHA at 0x8c5D54B2CAA4C2D08B0DDF82a1e6D2641779B8EC is verified
- ✅ Contract onKeyPurchaseHook_GUILD at 0xaE8F3F0826A39122401ED634f0a5C19549331432 is verified
- ✅ Subgraph up and running
- ✅ Unlock ownership correctly set to the multisig

We have found the following errors :

- ❌ Verification failed for onKeyPurchaseHook_PROMO_CODE_CAPPED at 0x4Bf912519549DF750002814a2DcE7184b3971F06: Contract source code not verified
- ❌ Unlock source code at 0x70B3c9Dd9788570FAAb24B92c3a57d99f8186Cc7 does not match packaged UnlockV13
- ❌ PublicLock source code at 0x70cBE5F72dD85aA634d07d2227a421144Af734b3 does not match packaged PublicLockV14
- ❌ Missing signers: 0x2785f2a3DDaCfDE5947F1A9D6c878CCD7F885400,0x7A23608a8eBe71868013BDA0d900351A83bb4Dc2,0x8de33D8204929ceb2F7AA6299d0643a7f6664c9b




Some additonal warnings  :

- ⚠️ Contract PublicLockPrevious is missing
- ⚠️ Contract onKeyPurchaseHook_CUSTOM_CONTRACT is missing
- ⚠️ Contract onKeyPurchaseHook_GITCOIN is missing
- ⚠️ Contract onKeyPurchaseHook_ADVANCED_TOKEN_URI is missing

**Comment by unlock-deployer** [comment]
### packages/networks/src/networks/linea.ts

The setup is successful :

- ✅ all mandatory keys are present
- ✅ Contract PublicLockLatest at 0x70cBE5F72dD85aA634d07d2227a421144Af734b3 is verified
- ✅ Contract Unlock at 0x70B3c9Dd9788570FAAb24B92c3a57d99f8186Cc7 is verified
- ✅ Contract KeyManager at 0x338b1f296217485bf4df6CE9f93ab4C73F72b57D is verified
- ✅ Contract onKeyPurchaseHook_PASSWORD at 0x6878Ae3c863f6Ebd27B47C02F6B32aAC8B0BA07E is verified
- ✅ Contract onKeyPurchaseHook_PROMOCODE at 0xCD9C9b40D757b56359e19563203D3bc64089638d is verified
- ✅ Contract onKeyPurchaseHook_PASSWORD_CAPPED at 0xD925Ac2887Ba4372849F0fd64217A6749552bb21 is verified
- ✅ Contract onKeyPurchaseHook_CAPTCHA at 0x8c5D54B2CAA4C2D08B0DDF82a1e6D2641779B8EC is verified
- ✅ Contract onKeyPurchaseHook_GUILD at 0xaE8F3F0826A39122401ED634f0a5C19549331432 is verified
- ✅ Subgraph up and running
- ✅ Unlock ownership correctly set to the multisig

We have found the following errors :

- ❌ Verification failed for onKeyPurchaseHook_PROMO_CODE_CAPPED at 0x4Bf912519549DF750002814a2DcE7184b3971F06: Contract source code not verified
- ❌ Unlock source code at 0x70B3c9Dd9788570FAAb24B92c3a57d99f8186Cc7 does not match packaged UnlockV13
- ❌ PublicLock source code at 0x70cBE5F72dD85aA634d07d2227a421144Af734b3 does not match packaged PublicLockV14
- ❌ Missing signers: 0x2785f2a3DDaCfDE5947F1A9D6c878CCD7F885400,0x7A23608a8eBe71868013BDA0d900351A83bb4Dc2,0x8de33D8204929ceb2F7AA6299d0643a7f6664c9b




Some additonal warnings  :

- ⚠️ Contract PublicLockPrevious is missing
- ⚠️ Contract onKeyPurchaseHook_CUSTOM_CONTRACT is missing
- ⚠️ Contract onKeyPurchaseHook_GITCOIN is missing
- ⚠️ Contract onKeyPurchaseHook_ADVANCED_TOKEN_URI is missing

**Comment by unlock-deployer** [comment]
### packages/networks/src/networks/linea.ts

The setup is successful :

- ✅ all mandatory keys are present
- ✅ Contract PublicLockLatest at 0x4A553635774b3d6fB6273A83b5B49577dF450227 is verified
- ✅ Contract PublicLockPrevious at 0x70cBE5F72dD85aA634d07d2227a421144Af734b3 is verified
- ✅ Contract Unlock at 0x70B3c9Dd9788570FAAb24B92c3a57d99f8186Cc7 is verified
- ✅ Contract KeyManager at 0x338b1f296217485bf4df6CE9f93ab4C73F72b57D is verified
- ✅ Contract onKeyPurchaseHook_PASSWORD at 0x6878Ae3c863f6Ebd27B47C02F6B32aAC8B0BA07E is verified
- ✅ Contract onKeyPurchaseHook_PROMOCODE at 0xCD9C9b40D757b56359e19563203D3bc64089638d is verified
- ✅ Contract onKeyPurchaseHook_PASSWORD_CAPPED at 0xD925Ac2887Ba4372849F0fd64217A6749552bb21 is verified
- ✅ Contract onKeyPurchaseHook_CAPTCHA at 0x8c5D54B2CAA4C2D08B0DDF82a1e6D2641779B8EC is verified
- ✅ Contract onKeyPurchaseHook_GUILD at 0xaE8F3F0826A39122401ED634f0a5C19549331432 is verified
- ✅ Unlock source code at 0x70B3c9Dd9788570FAAb24B92c3a57d99f8186Cc7 matches packaged UnlockV13
- ✅ PublicLock source code at 0x4A553635774b3d6fB6273A83b5B49577dF450227 matches packaged PublicLockV14
- ✅ Subgraph up and running
- ✅ Unlock ownership correctly set to the multisig

We have found the following errors :

- ❌ Verification failed for onKeyPurchaseHook_PROMO_CODE_CAPPED at 0x4Bf912519549DF750002814a2DcE7184b3971F06: Contract source code not verified
- ❌ Missing signers: 0x2785f2a3DDaCfDE5947F1A9D6c878CCD7F885400,0x7A23608a8eBe71868013BDA0d900351A83bb4Dc2,0x8de33D8204929ceb2F7AA6299d0643a7f6664c9b




Some additonal warnings  :

- ⚠️ Contract onKeyPurchaseHook_CUSTOM_CONTRACT is missing
- ⚠️ Contract onKeyPurchaseHook_GITCOIN is missing
- ⚠️ Contract onKeyPurchaseHook_ADVANCED_TOKEN_URI is missing

File: packages/networks/src/networks/linea.ts [hunk 1]
```diff
@@ -72,7 +80,7 @@ export const linea: NetworkConfig = {
     symbol: 'ETH',
   },
   previousDeploys: [],
-  provider: 'https://rpc.unlock-protocol.com/59144',
```
Comment (julien51) [inline packages/networks/src/networks/linea.ts:75]: this is right tho!

File: packages/networks/src/networks/linea.ts [hunk 2]
```diff
@@ -72,7 +80,7 @@ export const linea: NetworkConfig = {
...
+  provider: 'https://rpc.linea.build/',
```
Comment (clemsos) [inline packages/networks/src/networks/linea.ts:83]:
```suggestion
  provider: 'https://rpc.unlock-protocol.com/59144',
```

**Comment by unlock-deployer** [comment]
### packages/networks/src/networks/linea.ts

The setup is successful :

- ✅ all mandatory keys are present
- ✅ Contract PublicLockLatest at 0x4A553635774b3d6fB6273A83b5B49577dF450227 is verified
- ✅ Contract PublicLockPrevious at 0x70cBE5F72dD85aA634d07d2227a421144Af734b3 is verified
- ✅ Contract Unlock at 0x70B3c9Dd9788570FAAb24B92c3a57d99f8186Cc7 is verified
- ✅ Contract KeyManager at 0x338b1f296217485bf4df6CE9f93ab4C73F72b57D is verified
- ✅ Contract onKeyPurchaseHook_PASSWORD at 0x6878Ae3c863f6Ebd27B47C02F6B32aAC8B0BA07E is verified
- ✅ Contract onKeyPurchaseHook_PROMOCODE at 0xCD9C9b40D757b56359e19563203D3bc64089638d is verified
- ✅ Contract onKeyPurchaseHook_PASSWORD_CAPPED at 0xD925Ac2887Ba4372849F0fd64217A6749552bb21 is verified
- ✅ Contract onKeyPurchaseHook_CAPTCHA at 0x8c5D54B2CAA4C2D08B0DDF82a1e6D2641779B8EC is verified
- ✅ Contract onKeyPurchaseHook_GUILD at 0xaE8F3F0826A39122401ED634f0a5C19549331432 is verified
- ✅ Unlock source code at 0x70B3c9Dd9788570FAAb24B92c3a57d99f8186Cc7 matches packaged UnlockV13
- ✅ PublicLock source code at 0x4A553635774b3d6fB6273A83b5B49577dF450227 matches packaged PublicLockV14
- ✅ Subgraph up and running
- ✅ Unlock ownership correctly set to the multisig

We have found the following errors :

- ❌ Verification failed for onKeyPurchaseHook_PROMO_CODE_CAPPED at 0x4Bf912519549DF750002814a2DcE7184b3971F06: Contract source code not verified




Some additonal warnings  :

- ⚠️ Contract onKeyPurchaseHook_CUSTOM_CONTRACT is missing
- ⚠️ Contract onKeyPurchaseHook_GITCOIN is missing
- ⚠️ Contract onKeyPurchaseHook_ADVANCED_TOKEN_URI is missing

**Comment by clemsos** [comment]
merge once #14665

**Comment by unlock-deployer** [comment]
### packages/networks/src/networks/linea.ts

The setup is successful :

- ✅ all mandatory keys are present
- ✅ Contract PublicLockLatest at 0x4A553635774b3d6fB6273A83b5B49577dF450227 is verified
- ✅ Contract PublicLockPrevious at 0x70cBE5F72dD85aA634d07d2227a421144Af734b3 is verified
- ✅ Contract Unlock at 0x70B3c9Dd9788570FAAb24B92c3a57d99f8186Cc7 is verified
- ✅ Contract KeyManager at 0x338b1f296217485bf4df6CE9f93ab4C73F72b57D is verified
- ✅ Contract onKeyPurchaseHook_PASSWORD at 0x6878Ae3c863f6Ebd27B47C02F6B32aAC8B0BA07E is verified
- ✅ Contract onKeyPurchaseHook_PROMOCODE at 0xCD9C9b40D757b56359e19563203D3bc64089638d is verified
- ✅ Contract onKeyPurchaseHook_PROMO_CODE_CAPPED at 0x04664b4290fa1F4001ED25d9576f7C2d980aC64d is verified
- ✅ Contract onKeyPurchaseHook_PASSWORD_CAPPED at 0xD925Ac2887Ba4372849F0fd64217A6749552bb21 is verified
- ✅ Contract onKeyPurchaseHook_CAPTCHA at 0x8c5D54B2CAA4C2D08B0DDF82a1e6D2641779B8EC is verified
- ✅ Contract onKeyPurchaseHook_GUILD at 0xaE8F3F0826A39122401ED634f0a5C19549331432 is verified
- ✅ Unlock source code at 0x70B3c9Dd9788570FAAb24B92c3a57d99f8186Cc7 matches packaged UnlockV13
- ✅ PublicLock source code at 0x4A553635774b3d6fB6273A83b5B49577dF450227 matches packaged PublicLockV14
- ✅ Subgraph up and running
- ✅ Unlock ownership correctly set to the multisig







Some additonal warnings  :

- ⚠️ Contract onKeyPurchaseHook_CUSTOM_CONTRACT is missing
- ⚠️ Contract onKeyPurchaseHook_GITCOIN is missing
- ⚠️ Contract onKeyPurchaseHook_ADVANCED_TOKEN_URI is missing

**Comment by julien51** [comment]
ok, looking good! I think we just need to verify the delayMod ?

**Comment by clemsos** [comment]
> ok, looking good! I think we just need to verify the delayMod ?

yes asking again to the Gnosis Guild team

**Comment by unlock-deployer** [comment]
### packages/networks/src/networks/linea.ts

The setup is successful :

- ✅ all mandatory keys are present
- ✅ Contract PublicLockLatest at 0x4A553635774b3d6fB6273A83b5B49577dF450227 is verified
- ✅ Contract PublicLockPrevious at 0x70cBE5F72dD85aA634d07d2227a421144Af734b3 is verified
- ✅ Contract Unlock at 0x70B3c9Dd9788570FAAb24B92c3a57d99f8186Cc7 is verified
- ✅ Contract KeyManager at 0x338b1f296217485bf4df6CE9f93ab4C73F72b57D is verified
- ✅ Contract onKeyPurchaseHook_PASSWORD at 0x6878Ae3c863f6Ebd27B47C02F6B32aAC8B0BA07E is verified
- ✅ Contract onKeyPurchaseHook_PROMOCODE at 0xCD9C9b40D757b56359e19563203D3bc64089638d is verified
- ✅ Contract onKeyPurchaseHook_PROMO_CODE_CAPPED at 0x04664b4290fa1F4001ED25d9576f7C2d980aC64d is verified
- ✅ Contract onKeyPurchaseHook_PASSWORD_CAPPED at 0xD925Ac2887Ba4372849F0fd64217A6749552bb21 is verified
- ✅ Contract onKeyPurchaseHook_CAPTCHA at 0x8c5D54B2CAA4C2D08B0DDF82a1e6D2641779B8EC is verified
- ✅ Contract onKeyPurchaseHook_GUILD at 0xaE8F3F0826A39122401ED634f0a5C19549331432 is verified
- ✅ Unlock source code at 0x70B3c9Dd9788570FAAb24B92c3a57d99f8186Cc7 matches packaged UnlockV13
- ✅ PublicLock source code at 0x4A553635774b3d6fB6273A83b5B49577dF450227 matches packaged PublicLockV14
- ✅ Subgraph up and running
- ✅ Unlock ownership correctly set to the multisig







Some additonal warnings  :

- ⚠️ Contract onKeyPurchaseHook_CUSTOM_CONTRACT is missing
- ⚠️ Contract onKeyPurchaseHook_GITCOIN is missing
- ⚠️ Contract onKeyPurchaseHook_ADVANCED_TOKEN_URI is missing

**Comment by clemsos** [comment]
ok the module is now verified, thanks to @auryn-macmillan  https://lineascan.build/address/0xd54895B1121A2eE3f37b502F507631FA1331BED6#code

**Comment by unlock-deployer** [comment]
### packages/networks/src/networks/linea.ts

The setup is successful :

- ✅ all mandatory keys are present
- ✅ Subgraph up and running
- ✅ Unlock ownership correctly set to the multisig
- ✅ Contract PublicLockLatest at 0x4A553635774b3d6fB6273A83b5B49577dF450227 is verified
- ✅ Contract PublicLockPrevious at 0x70cBE5F72dD85aA634d07d2227a421144Af734b3 is verified
- ✅ Contract Unlock at 0x70B3c9Dd9788570FAAb24B92c3a57d99f8186Cc7 is verified
- ✅ Contract KeyManager at 0x338b1f296217485bf4df6CE9f93ab4C73F72b57D is verified
- ✅ Contract onKeyPurchaseHook_PASSWORD at 0x6878Ae3c863f6Ebd27B47C02F6B32aAC8B0BA07E is verified
- ✅ Contract onKeyPurchaseHook_PROMOCODE at 0xCD9C9b40D757b56359e19563203D3bc64089638d is verified
- ✅ Contract onKeyPurchaseHook_PROMO_CODE_CAPPED at 0x04664b4290fa1F4001ED25d9576f7C2d980aC64d is verified
- ✅ Contract onKeyPurchaseHook_PASSWORD_CAPPED at 0xD925Ac2887Ba4372849F0fd64217A6749552bb21 is verified
- ✅ Contract onKeyPurchaseHook_CAPTCHA at 0x8c5D54B2CAA4C2D08B0DDF82a1e6D2641779B8EC is verified
- ✅ Contract onKeyPurchaseHook_GUILD at 0xaE8F3F0826A39122401ED634f0a5C19549331432 is verified
- ✅ Unlock source code at 0x70B3c9Dd9788570FAAb24B92c3a57d99f8186Cc7 matches packaged UnlockV13
- ✅ PublicLock source code at 0x4A553635774b3d6fB6273A83b5B49577dF450227 matches packaged PublicLockV14







Some additonal warnings  :

- ⚠️ Contract onKeyPurchaseHook_CUSTOM_CONTRACT is missing
- ⚠️ Contract onKeyPurchaseHook_GITCOIN is missing
- ⚠️ Contract onKeyPurchaseHook_ADVANCED_TOKEN_URI is missing

**Comment by unlock-deployer** [comment]
### packages/networks/src/networks/linea.ts

The setup is successful :

- ✅ all mandatory keys are present
- ✅ Subgraph up and running
- ✅ Unlock ownership correctly set to the multisig
- ✅ Contract PublicLockLatest at 0x4A553635774b3d6fB6273A83b5B49577dF450227 is verified
- ✅ Contract PublicLockPrevious at 0x70cBE5F72dD85aA634d07d2227a421144Af734b3 is verified
- ✅ Contract Unlock at 0x70B3c9Dd9788570FAAb24B92c3a57d99f8186Cc7 is verified
- ✅ Contract KeyManager at 0x338b1f296217485bf4df6CE9f93ab4C73F72b57D is verified
- ✅ Contract onKeyPurchaseHook_PASSWORD at 0x6878Ae3c863f6Ebd27B47C02F6B32aAC8B0BA07E is verified
- ✅ Contract onKeyPurchaseHook_PROMOCODE at 0xCD9C9b40D757b56359e19563203D3bc64089638d is verified
- ✅ Contract onKeyPurchaseHook_PROMO_CODE_CAPPED at 0x04664b4290fa1F4001ED25d9576f7C2d980aC64d is verified
- ✅ Contract onKeyPurchaseHook_PASSWORD_CAPPED at 0xD925Ac2887Ba4372849F0fd64217A6749552bb21 is verified
- ✅ Contract onKeyPurchaseHook_CAPTCHA at 0x8c5D54B2CAA4C2D08B0DDF82a1e6D2641779B8EC is verified
- ✅ Contract onKeyPurchaseHook_GUILD at 0xaE8F3F0826A39122401ED634f0a5C19549331432 is verified
- ✅ Unlock source code at 0x70B3c9Dd9788570FAAb24B92c3a57d99f8186Cc7 matches packaged UnlockV13
- ✅ PublicLock source code at 0x4A553635774b3d6fB6273A83b5B49577dF450227 matches packaged PublicLockV14







Some additonal warnings  :

- ⚠️ Contract onKeyPurchaseHook_CUSTOM_CONTRACT is missing
- ⚠️ Contract onKeyPurchaseHook_GITCOIN is missing
- ⚠️ Contract onKeyPurchaseHook_ADVANCED_TOKEN_URI is missing

**Review by julien51 — APPROVED** [review]

**Pull request merged**